Sources/Build/BuildPlan/BuildPlan.swift

```diff
@@ -216,6 +216,10 @@ public class BuildPlan: SPMBuildCore.BuildPlan {
         AnySequence(self.productMap.values.map { $0 as SPMBuildCore.ProductBuildDescription })
     }
 
+    public var buildModules: AnySequence<SPMBuildCore.ModuleBuildDescription> {
```

Comment:
would this be better as `some Sequence<T>`

Reply:
The other member uses AnySequence so I used the same for consistency.